docs-library/10x-Governance.md

```diff
@@ -0,0 +1,181 @@
+
+As a project advances through the phases, the project team must show that they have identified and addressed the types of project risks typical to projects at each phase. Alongside the project team, the 10x team is an active participant in helping teams identify risks associated with their project at each stage, and in ensuring that teams are able to successfully identify how these risks can best be addressed.
+
+##What criteria are used to fund 10x projects?
+As part of determining whether or not a product or service meets a real, verifiable need, 10x puts a significant emphasis on funding exploration and development of products and services that meet one or more of the following criteria:
+
+* **Proactive support for upcoming or recently implemented law and policy**. As technology and related services have become more central to successfully enacting new laws and regulations, OPP is frequently called upon to provide implementation support for new or upcoming laws or policies. 10x actively seeks to find and fund new projects in areas of law and policy where we either have been or believe we may be asked to provide implementation support.
+
+* **Improved support for existing law or policy**. 10x funds products or services that can demonstrate how the application of technology can meaningfully improve understanding and compliance with law and regulation.
+
+* **Technology or process to make law, policy, or security compliance more effective and cost-efficient**. Recent technology developments can be deployed to improve the effectiveness of existing government services. 10x looks for areas to invest in where implementation of new technologies can have an outsize impact on the function or success of a product or service offered by the federal government.
+
+* **Bringing best practices from the private sector into government**. Products and services that bring all the best of new technology and invention into the government environment. These projects focus not just on technical implementation but also do significant work to ensure that as these technologies are deployed they are fully compliant with all relevant law and policy.
+
+
+## First approval: From submission to Phase 1 Investigation
+
+Any project selected for 10x funding begins at the “Investigation” phase, a one to two week long research sprint. The goal of this first phase is to identify whether or not there are any significant blockers to the actively pursuing the project. Are there any technological hurdles that could keep the project from being successful? Are there regulatory restrictions that are too much of a challenge to overcome? Can the team locate any reason that the federal marketplace may not yet be ready to make use of the product or service proposed, or that it’s unlikely that federal agencies will be willing to pay for the product if it’s intended to be cost-recoverable?
+
+**Criteria:**
```

Comment:
I'd make this an h3 but otherwise good document structure in the headings throughout

Reply:
Actually looking again maybe I wouldn't – but it doesn't seem consistent with the headings within the section. 🤔